### DISCUSSION TITLE
send data to specific client server-end

### DISCUSSION THREAD

**Discussion by cherimo** [socketio/socket.io#4085] [Q&A]
I can't find any clear information on the internet what the comparable function should be of the following code:

```
socket.to(socketClient).emit("eventName", "Message", function(res) {
   console.log("Respont: ",res);
})
```
I really need something like this where I can send Data to the Server-side of the client instead of the Client-side.

Here https://newbedev.com/socket-io-emit-to-socket-id-code-example they indicate the following text:

// WARNING: socket.to(socket.id).emit() will NOT work, as it will send to everyone in the room named socket.id but the sender. Please use the classic socket.emit() instead.

Is there any alternative to this?

**Comment by darrachequesne** [comment]
Hi! You can use the `serverSideEmit` functionality for this use case: https://socket.io/docs/v4/server-instance/#serversideemit
